### PR TITLE
Aragon Flock Proposal: PanAragon

### PR DESCRIPTION
**Pull request by apeir99n**
PanAragon is the “(un)bank-like” arm of Aragon. PanAragon is network of DAOs inspired by decentralised financial systems, aiming to find almost immediate integrations with real world use-cases.

We are the PanAragon team, and we have a proposal for the community and the Board.

We are aware that we are slightly late to the Flock party, but we (PanAragon) have been working and researching for this proposal for a long time, and only started formalizing some weeks ago.

We believe the community should judge the proposal for the value it brings to the Aragon ecosystem. We don’t want to disrupt the flow and/or any processes. We just achieved our goals in terms of proposal development in a shorter time than we expected it and would like to push for radical transparency by just posting it, and seeing how the community sees what we envision.
We are impressed by the feedback rounds from previous Flock proposals, and since this particular one includes an element of monetization for the network, are very curious to see how this sinks in. So read it, send us comments, roast us, everything works - **we are really confident the PanAragon proposal is tailored for what, to the best of our knowledge, this project and community is about** :slightly_smiling_face:

Proposal TLDR: In layman terms: if Aragon is a country, PanAragon this would be the Ministry of Finance, but as well, they would be the direct interface with real world use cases to learn from them. We also propose a monetisation model able to help the network and to “pay back” the grant money, an R&D arm, cross-collaboration with teams, and of course, dig into the technicalities.

[Read the Full Proposal →](https://github.com/akropolisio/flock/blob/master/teams/PanAragon/2019.md)

### CODE REVIEW

**Comment by semuelle** [comment]
> AragonOS is a technically complex product and it consumes a large amount of gas, 150,000+ units of gas

For what? Per transaction? Any transaction?

**Comment by semuelle** [comment]
With regard to adoption I would recommend offering more interfaces than a standalone mobile client. Could this not be integrated with the Aragon mobile app? Or consider building a solution for the less-tech-savvy, like so: https://devpost.com/software/dao-to-go.

**Comment by apeir99n** [comment]
> > AragonOS is a technically complex product and it consumes a large amount of gas, 150,000+ units of gas
> 
> For what? Per transaction? Any transaction?

it's DAO's deploying

> With regard to adoption I would recommend offering more interfaces than a standalone mobile client. Could this not be integrated with the Aragon mobile app? Or consider building a solution for the less-tech-savvy, like so: https://devpost.com/software/dao-to-go.

thanks for feedback. 
Our target market is Kenya, where people are active users of mPesa as a payment service. Therefore, we decided to focus on mobile. However, we will also do the web version.

**Comment by stefanobernardi** [comment]
Closing after forum feedback and decision of the team to change proposal for a Nest grant
